pkgs/development/python-modules/cleo/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi,
+  pylev, pastel, clikit }:
+buildPythonPackage rec {
+  pname = "cleo";
+  version = "0.7.2";
+
+  src = fetchPypi {
```

Comment:
No tests in this tarball either :frowning:

Reply:
Also wants to build using `poetry`.